views/webpack.config.js

```diff
@@ -0,0 +1,177 @@
+                        query: 'Bloodhound'
+                    }
+                ]
+            }, {
+                test: /dropzone\/dist\/dropzone\.js/,
+                loader: 'imports-loader?this=>window&module=>null'
+            }, {
+                test: require.resolve('moment'),
+                loader: 'imports-loader?define=>false&this=>window',
+            }, {
+                test: /typeahead\.jquery\.js/,
+                loader: 'imports-loader?define=>false&exports=>false&this=>window'
+            }, {
+                test: /bootstrap-tokenfield\.js/,
+                loader: 'imports-loader?define=>false&exports=>false&this=>window'
+            }, {
+                test: /bootstrap-datetimepicker\.js/,
+                loader: 'imports-loader?define=>false&exports=>false&this=>window'
+            }, {
+                test: /jwerty\/jwerty\.js/,
+                loader: 'imports-loader?this=>window&module=>false'
+            }, {
+                test: /\.vue$/,
+                loader: 'vue-loader',
+                options: {
+                    loaders: {
+                        js: 'babel-loader?presets[]=es2015&presets[]=stage-2',
+                        css: 'postcss-loader'
+                    },
+                }
+            }, {
+                test: /\.css$/,
+                use: ExtractTextPlugin.extract({
+                    fallback: 'style-loader',
+                    use: ['css-loader']
+                })
+            }, {
+                test: /\.scss$/,
+                use: ExtractTextPlugin.extract({
+                    use: [
+                        {
+                            loader: 'css-loader',
+                            options: {
+                                minimize: true,
+                                //sourceMap: true, // uncomment me to generate source maps
+                            }
+                        },
+                        {
+                            loader: 'postcss-loader',
+                            options: {
+                                //sourceMap: true, // uncomment me to generate source maps
+                            }
+                        },
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                //sourceMap: true, // uncomment me to generate source maps
+                            }
+                        }
+                    ]
+                })
+            }, {
+                test: /.(jpg|png|woff(2)?|eot|otf|ttf|svg|gif)(\?[a-z0-9=\.]+)?$/,
+                use: 'file-loader?name=[hash].[ext]'
+            }
+        ]
+    },
+    plugins: [
+        new ExtractTextPlugin('theme.css'),
+        new webpack.ProvidePlugin({
+            moment: 'moment', // needed for bootstrap datetime picker
+        })
+    ]
+};
+
+if (process.env.NODE_ENV === 'production') {
+    /*config.plugins.push(
```

Comment:
Hum no need to comment this parts

Reply:
it didn't work with the plugin enabled so I commented it to avoid losing time, but I know I need to fix it and build prod assets